src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -1630,4 +1686,17 @@ private void logAttackMoves(final List<ProTerritory> prioritizedTerritories) {
       }
     }
   }
+
+  private boolean canAirSafelyLandAfterAttack(final Unit unit, final Territory t) {
+    final boolean isAdjacentToAlliedFactory =
+        Matches.territoryHasNeighborMatching(data,
+            ProMatches.territoryHasInfraFactoryAndIsAlliedLand(player, data)).match(t);
+    final int range = TripleAUnit.get(unit).getMovementLeft();
+    final int distance =
+        data.getMap().getDistance_IgnoreEndForCondition(ProData.unitTerritoryMap.get(unit), t,
```

Comment:
That method is only called by AI code.The only reason to touch it, or any variation of it, is if you were modifying AI code. Adding a second API that is simply removes the one boolean param, and calls the original method with a value of true, will always be exactly 3 lines of code. Here it is for you:

```
 public static Match<Territory> territoryCanMoveAirUnitsAndNoAA(final PlayerID player, final GameData data) {
  return territoryCanMoveAirUnitsAndNoAA(player, data);
}
```